packages/mantine/src/components/prompt/Prompt.tsx

```diff
@@ -0,0 +1,64 @@
+import {createStyles, Modal, ModalProps} from '@mantine/core';
+import {Children, ReactElement, ReactNode} from 'react';
+import {PromptFooter} from './PromptFooter';
+
+const useStyles = createStyles((theme) => {
+    const white = '#fff';
+    return {
+        body: {
+            padding: 0,
+        },
+        modalType: {overflow: 'hidden', width: 550},
+        innerBody: {
+            padding: `${theme.spacing.md}px ${theme.spacing.xl}px ${theme.spacing.lg}px`,
+        },
+        header: {
+            padding: `${theme.spacing.md}px ${theme.spacing.xl}px`,
+            width: '100%',
+            borderBottom: `1px solid ${theme.colors.gray[3]}`,
+            fontSize: theme.headings.sizes.h4.fontSize,
+            lineHeight: theme.headings.sizes.h4.fontSize,
+        },
+        success: {backgroundColor: theme.colors.lime[6], color: white},
+        warning: {backgroundColor: theme.colors.yellow[5], color: white},
+        critical: {
+            backgroundColor: theme.colors.red[6],
+            color: white,
+        },
+        info: {backgroundColor: theme.colors.navy[5], color: white},
+        whiteClose: {color: white, '&:hover': {backgroundColor: 'transparent'}},
+    };
+});
+
+export interface PromptProps extends ModalProps {
+    variant?: 'success' | 'warning' | 'critical' | 'info';
+    children: ReactNode;
+}
+interface PromptType {
+    (props: PromptProps): ReactElement;
+    Footer: typeof PromptFooter;
+}
+
+export const Prompt: PromptType = ({children, variant, size, ...otherProps}) => {
```

Comment:
I think now that we renamed the component to prompt, we can get rid of the "white" variant, since we probably should use a regular mantine modal in this case, what do you think?

```suggestion
export const Prompt: PromptType = ({children, variant = 'info', size, ...otherProps}) => {
```

Reply:
I'm not sure I fully understand your question, but the use-case I worry about is when someone wants to have a complex modal with tabs in it, and perhaps other suff, we cannot enforce having padding, a footer and a header.

Reply:
Okay, well in that case lets keep the "simple" version, but add a variant for it. That way it'll be clear that we support this variant